microshift_getting_started/microshift-architecture.adoc

```diff
@@ -19,13 +36,30 @@ Learn the specifics of {product-title} architecture including design intent, how
 * Compatible with RPM-OStree-based systems
 * Uses only the APIs needed for essential functions, such as security and runtime controls
 * Enables a subset of commands from the OpenShift CLI (`oc`) tool
+* Does not support high availability (HA) by the adding of worker nodes
```

Comment:
I can't comment on the line in this PR, but I'm not quite sure what "Application-level events and metrics are observation-only" means.

Reply:
It means no admin is done, just observation.